### PR TITLE
Added additional Nvida GPUs

### PR DESCRIPTION
**Pull request by vexr**
- Added Volta architecture as the baseline for NVIDIA GPUs and included Hopper as a supported architecture.

- Addressed various formatting and grammar issues.

### CODE REVIEW

File: docs/farming-&-staking/farming/additional-guides/gpu-plotter.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Although GPU plotting is not mandatory, it provides enhanced energy efficiency a
 
 
 :::note Plot Format Compatibility
-GPU plotting employs the new **v1 plot format**, which is applicable to any plots created with versions released on or after **July 5th**. In contrast, older software versions generated plots in the **v0 format**, which is only compatible with CPU plotting.
+GPU plotting employs the new v1 plot format, which is applicable to any plots created with versions released on or after uly 5th. In contrast, older software versions generated plots in the v0 format, which is only compatible with CPU plotting.
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/additional-guides/gpu-plotter.md:21]:
```suggestion
GPU plotting employs the new v1 plot format, which is applicable to any plots created with versions released on or after July 5th. In contrast, older software versions generated plots in the v0 format, which is only compatible with CPU plotting.
```

File: docs/farming-&-staking/farming/additional-guides/gpu-plotter.md [hunk 2]
```diff
@@ -31,45 +31,48 @@ GPU plotting employs the new **v1 plot format**, which is applicable to any plot
...
-|-------------------|---------------------------------------------|
-| RTX 40 Series     | RTX 4060, RTX 4060 Ti, RTX 4070, RTX 4070 Ti, RTX 4080, RTX 4090 |
-| RTX 30 Series     | RTX 3060, RTX 3060 Ti, RTX 3070, RTX 3070 Ti, RTX 3080, RTX 3080 Ti, RTX 3090, RTX 3090 Ti |
-| RTX 20 Series     | RTX 2060, RTX 2060 Super, RTX 2070, RTX 2070 Super, RTX 2080, RTX 2080 Super, RTX 2080 Ti |
-| Drivers           | Nvidia drivers version **550** or later    |
+| Architecture      | Models                                                                 |
+|-------------------|------------------------------------------------------------------------|
+| Hopper            | H Series (H100, H200, H300)                                            |
+| Ada Lovelace      | RTX 40 Series                                                          |
+| Ampere            | RTX 30 Series <br /> A Series (A100, A40, A30, A10, A6000, A5000, A4000) |
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/additional-guides/gpu-plotter.md:41]:
```suggestion
| Ampere            | RTX 30 Series A Series (A100, A40, A30, A10, A6000, A5000, A4000) |
```

File: docs/farming-&-staking/farming/additional-guides/gpu-plotter.md [hunk 3]
```diff
@@ -31,45 +31,48 @@ GPU plotting employs the new **v1 plot format**, which is applicable to any plot
...
+| Turing            | RTX 20 Series <br /> Quadro RTX Series (RTX 8000, RTX 6000)              |
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/additional-guides/gpu-plotter.md:42]:
```suggestion
| Turing            | RTX 20 Series Quadro RTX Series (RTX 8000, RTX 6000)              |
```

**Comment by EmilFattakhov** [comment]
Hope you don't mind me experimenting and committing to your branch :) Turns out you can't use <br /> inside the table, which I also wasn't aware of

**Comment by vexr** [comment]
Attempting `&#10;` for line breaks. If not, a single line for each architecture will have to do.

**Review by EmilFattakhov — APPROVED** [review]
Thanks for your commitment @vexr 🚀

**Pull request merged**